sdk/OoyalaSkinSDK/widgets/ResponsiveList.js

```diff
@@ -67,20 +67,25 @@ var ResponsiveList = React.createClass({
 
   renderSlice: function(slice: object, i: number) {
     var sliceStyle = this.props.horizontal ? styles.column : styles.row;
+
+    renderItem = this.renderItem;
```

Comment:
I did that because if we call 'this' inside the slice.map context, we can't call the defined functions from the ResponsiveList. So I make the renderItem a variable so I can access it within slice.map.